src/collections/blog/2020-09-25-announcing-meshmates/index.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: "Announcing a new line of MeshMates"
+date: 2020-09-25 08:00:00 +0000
+author: our MeshMates with 🤍
+thumbnail: ./meshmate-announcement.png
+tags:
+ - Community
+ - Projects
+ - News
+published: true
+---
+
+import Ruth from "./meshmate-ruth-ikegah.png";
+import Kelechi from "./meshmate-kelechi-precious.png";
+import Tanuj from "./meshmate-tanuj-agarwal.png";
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+
+  _[Layer5 MeshMates](/community/meshmates) are committed to helping community members be successful contributors. MeshMates aid in identifying areas of projects to engage within, working groups to join, and in helping community members grow in their open source and cloud native knowledge. By connecting one-on-one, MeshMates will share tips on how to have the best community experience possible._
+
+  <h4>
+    <a href="https://layer5.io/community/members/tanuj-agarwal">
+      Tanuj Agarwal
+    </a>
+  </h4>
+  <img
+    src={ Tanuj }
+    class="image-right"
+  />
+  <p>
+    I was introduced to the Layer5 community three months back. Being fairly new
+    to the world of open source and looking for new communities and technology to
+    delve into, I hopped onto the Layer5 bandwagon and was amazed by the warm
+    welcome I received from the wholesome community. I had done video animation
+    jobs on a small scale before, which is what led me to confidently take up the
+    task of creating a short intro jingle for the various community meetings
+    hosted on the Layer5 youtube channel. After iterating for about a week, I
+    finally came up with something worthwhile. And to my elation, it was accepted!
+    It still warms my heart to see the intro flash before every meeting.
+  </p>
+  <p>
+    Next up I started work on a next generation model of the layer5.io site. Being
+    a new project based on Gatsby and heavily relying on React, Javascript and
+    GraphQL, the site project was a perfect and challenging opportunity for me to
+    expand on my skill set. Speed learning my way through Gatsby, with a
+    continuous influx of help and helpful suggestions from my MeshMate, Kush
+    Trivedi and others within the community, I soon mastered the platform and have
+    spent weeks on creating countless issues and PRs in hopes of achieving the
+    project goals. A few months down the line, I started reviewing PRs and began
+    helping others contribute, something which I had never had the opportunity to
+    do before! After months of continuous effort, I am proud to write that I
+    recently joined as a maintainer on the project and was soon approached to be a
+    MeshMate, which I happily accepted. My learning graph has seen a soaring
+    positive tangent in the months I have been at Layer5 and I’m looking forward
+    to an even more educative and rewarding journey ahead 😊
+  </p>
+  <br />
+  <h4>
+    <a href="https://layer5.io/community/members/kelechi-nwachukwu">
```

Comment:
Change the link to layer5-ng website's member page

Reply:
like above